drv/ignition-api/src/lib.rs

```diff
@@ -52,6 +52,8 @@ pub enum IgnitionError {
     /// Indicates the given request conflicts with the Target system power
     /// state. Poll the Target state and retry if desired.
     RequestDiscarded,
+    #[idol(server_death)]
+    ServerRestarted,
```

Comment:
We already have `ServerDied = 1` up at the top; should that be our `#[idol(server_death)]` flavor?

Reply:
Almost certainly - nice catch!

Reply:
Done in d09d063